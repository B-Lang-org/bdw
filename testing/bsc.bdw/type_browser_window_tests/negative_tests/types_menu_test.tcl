
# Adds the specified type to the type hierarchy
handlers::add_type 

# Creates a new project
handlers::open_project project.bspec 

# Adds the specified type to the type hierarchy
handlers::add_type 

# Adds the specified type to the type hierarchy
handlers::add_type Tin

# Adds the specified type to the type hierarchy
handlers::remove_type

# Adds the specified type to the type hierarchy
handlers::add_type Type

# Adds the specified type to the type hierarchy
handlers::remove_type

# Adds the specified type to the type hierarchy
handlers::remove_type Tout

# Closes the current project
handlers::close_project

# Exits the workstation
exit
