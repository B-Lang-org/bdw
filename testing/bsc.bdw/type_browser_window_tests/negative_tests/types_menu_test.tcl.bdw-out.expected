There is no open project.
The type is not specified
Tin
The type is not specified
Type
The type is not specified
There is no type in the Type Browser window.
